### PR TITLE
Update and install PHP-CS-Fixer

### PR DESCRIPTION
**Pull request by toddy15**
Hi,

are you still using php-cs-fixer? If so, this PR would install it as a development dependency, cleaning up the obsolete filenames. If you then run `./vendor/bin/php-cs-fixer check`, there are many fixes available:

`Found 757 of 1383 files that can be fixed`

I've omitted those from this PR to keep it manageable. :smile:

Regards,
Tobias

### CODE REVIEW

File: .php-cs-fixer.dist.php [hunk 1]
```diff
@@ -2,10 +2,9 @@
 
 declare(strict_types=1);
 
-require_once __DIR__ . '/bin/dev/php-cs-fixer/vendor/autoload.php';
-
 $finder = PhpCsFixer\Finder::create()
 	->in(__DIR__)
+	->notPath('addons')
```
Comment (MrPetovan) [inline .php-cs-fixer.dist.php]: Shouldn't this be `addon` ?

**Review by MrPetovan — COMMENTED** [review]
I don't remember exactly why we stopped relying on PHP CS Fixer, but the "750 files can be fixed" may be an indication. Since the project didn't begin with PHP CS Fixer, it's hard to start using it mid-project because of the extensiveness of the changes.

Feel free to use it to fix the files you're working on, but please don't submit a patch with 700 changed files.

File: .php-cs-fixer.dist.php (hunk 1 above)
Comment (toddy15) [reply .php-cs-fixer.dist.php]: Yes, of course. I've force-pushed my branch, including this change.

**Comment by MrPetovan** [comment]
The automated code standards check fails with the following output:
```
+ export COMPOSER_HOME=.composer
+ ./bin/composer.phar run cs:install

                                                      
  [InvalidArgumentException]                          
  Script "cs:install" is not defined in this package  
                                                      

run-script [--timeout TIMEOUT] [--dev] [--no-dev] [-l|--list] [--] [<script>] [<args>]...
```
You removed this script from the Composer file but there still is a reference to it in the Woodpecker pipeline.

**Comment by toddy15** [comment]
Yes, unfortunately all those parts are connected in some way. I've missed the Docker CI. This should now be fixed.

My proposal would be as follows:

1.  Update PHP-CS-Fixer to the current version, using the standard location in `vendor/bin/php-cs-fixer`, keeping the old version at the old location `bin/dev/php-cs-fixer/vendor/bin/php-cs-fixer` in parallel (this PR)
2. Change https://git.friendi.ca/friendica/friendica-docker-ci to use this updated PHP-CS-Fixer in the standard location `vendor/bin/php-cs-fixer`
3. Remove the old scripts from composer.json

Apart from that, it seems to me that the script `.woodpecker/.code_standards_check.yml` does submit only added files to php-cs-fixer, not modified ones. Could this be the reason that there are ~700 files to fix? :smile:

Maybe @nupplaphil should take a look at this PR and add a comment if this approach is fine with him.

Regards,
Tobias

**Review by MrPetovan — APPROVED** [review]

**Comment by toddy15** [comment]
Created https://git.friendi.ca/friendica/friendica-docker-ci/pulls/16 for the second step.

**Pull request merged**